helm/fluent-logshipping-app/templates/configmap.yaml

```
@@ -67,27 +74,28 @@ data:
   inputs-syslog.conf: |
     [INPUT]
         Name              systemd
-        Path              /run/log/journal
         Tag               syslog.*
-        Parser            syslog
         DB                ${input_database_path}/syslog.db
         storage.type      {{ .Values.fluentbit.inputStorageTypes.syslog }}
         Mem_Buf_Limit     ${input_memory_buffer_limit}
         Strip_Underscores on
 
   inputs-auditing.conf: |
+    {{- if has "sshd" .Values.outputs.inputLogTypes }}
     # Get the sshd audit logs
     [INPUT]
         Name              systemd
         Tag               audit.ssh.*
-        Path              /run/log/journal
-        Parser            syslog
         DB                ${input_database_path}/audit_ssh.db
-        Systemd_Filter    SYSLOG_IDENTIFIER=sshd
-        Mem_Buf_Limit     ${input_memory_buffer_limit}
         storage.type      {{ .Values.fluentbit.inputStorageTypes.sshd }}
+        Mem_Buf_Limit     ${input_memory_buffer_limit}
         Strip_Underscores on
+        ## Filters journalctl logs for either the syslog identifier=sshd of the _SYSTEMD_UNIT=sshd.service to ensure we only get SSH logs.
```

I though this was an OR operator?

```suggestion
        ## Filters journalctl logs either by syslog identifier or systemd unit name to ensure we only get SSH logs.
```